docs/docs-beta/docs/getting-started/installation.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Installing Dagster"
+description: "Learn how to install Dagster"
+sidebar_position: 20
+sidebar_label: "Installation"
+---
+
+# Installing Dagster
+
+To follow the steps in this guide, you'll need:
+
+- To install Python 3.8 or higher. **Python 3.11 is recommended**.
```

Comment:
We should make add a todo to make version dependencies a component again, that was nice.

Reply:
Just did this and realized that, like other React-based SSGs, it renders partial components as block elements so they can't be used inline, ex:

<img width="401" alt="Screenshot 2024-09-06 at 10 52 40 AM" src="https://github.com/user-attachments/assets/8b8da84b-1a03-4e8b-bb4e-dbef02f0fe75">

Produced by:

```
To follow the steps in this guide, you'll need:

import DagsterPythonSupport from '../partials/\_DagsterPythonSupport.md';

- To install <DagsterPythonSupport />. **Python 3.11 is recommended**.
- To install pip, a Python package installer
```

I did some quick googling, but couldn't find an easy way to remove whitespace from components like this. We can restructure content so this is only on one line, but it would be nice to be able to use this inline when needed.

In Jekyll, for example, I could use a variable instead to do something like:

```
# _config.yml

python_support: "Python 3.8 or higher"


# some-content-file.md

Dagster supports {{ site.python_support }}.


# output

Dagster supports Python 3.8 or higher.
```